README.md

```diff
@@ -98,7 +105,7 @@ Further, we realized that this would allow us to build the terminal's renderer a
 
 ## I built and ran the new Terminal, but I just get a blank window app!
 
-Make sure you are building for your computer's architecture. If your box has a 64-bit Windows, change your Solution Platform to x64.  
+Make sure you are building for your computer's architecture. If your box has a 64-bit Windows, change your Solution Platform to x64.
```

Comment:
while you're changing this line, make the grammar more good.

Reply:
Done. Please re-check FAQ section. Improved it some more too.